docs/tutorials/fraud_online/1_feature_groups.ipynb

```diff
@@ -2,25 +2,15 @@
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "86f66405",
-   "metadata": {},
-   "source": [
-    "# Part 01: Load, Engineer & Connect\n",
-    "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/fraud_online/1_feature_groups.ipynb)\n",
+    "# <span style=\"font-width:bold; font-size: 3rem; color:#1EB182;\"><img src=\"../images/icon102.png\" width=\"38px\"></img> **Hopsworks Feature Store** </span><span style=\"font-width:bold; font-size: 3rem; color:#333;\">- Part 01: Load, Engineer & Connect</span>\n",
```

Comment:
Pretty sure this won't work. The image doesn't exist here. Also why all this HTML?